### PR TITLE
Add support for start listening

### PR DESCRIPTION
**Pull request by balloob**

### CODE REVIEW

File: zwave_js_server/client.py [hunk 1]
```diff
@@ -176,12 +188,23 @@ async def _handle_connection(self):
...
+                version_msg["serverVersion"],
+                version_msg["driverVersion"],
+            )
             self.state = STATE_CONNECTED
 
             if self._on_connect:
                 await gather_callbacks(self._logger, "on_connect", self._on_connect)
 
+            await client.send_json(
+                {"messageId": uuid.uuid4().hex, "command": "start_listening"}
```
Comment (marcelveldt) [inline zwave_js_server/client.py:205]: you might want to move creation of default id to the send_json method perhaps so it can be omitted ?

Comment (marcelveldt) [reply zwave_js_server/client.py:205]: but that means the send json needs to check for messageId key

**Review by MartinHjelmare — APPROVED** [review]
Python looks good.

File: zwave_js_server/client.py (hunk 1 above)
Comment (balloob) [reply zwave_js_server/client.py:205]: Will do that in the next pass when adding full support for commands.

**Pull request merged**